### PR TITLE
update(hedgedoc): bump version to 1.9.9 (includes security fix)

### PR DESCRIPTION
**Pull request by jdreichmann**
see https://github.com/hedgedoc/hedgedoc/security/advisories/GHSA-7494-7hcf-vxpg

### CODE REVIEW

**Review by jcgruenhage — APPROVED** [review]

**Comment by jcgruenhage** [comment]
Can you change the committer email, so that the CLA bot is happy, and upload your PGP key to github, so that they are happy too?

**Comment by jdreichmann** [comment]
> Can you change the committer email, so that the CLA bot is happy, and upload your PGP key to github, so that they are happy too?

My OpenPGP key is uploaded to Github, despite Github falsely claiming "This user has not yet uploaded their public signing key. ". Also i am using my normal committer email i use everytime/everywhere.

**Comment by jdreichmann** [comment]
Ansible lint seems to not run in progressive mode, blocking updates (incl security fixes) from being merged due to linting problems in unrelated roles.

**Comment by jcgruenhage** [comment]
you're using `Johanna Dorothea Reichmann <transcaffeine@finally.coffee>` as author information here, which causes github to assign that commit to @transcaffeine, which does not have the PGP key uploaded yet. It seems to me like you're trying to split between work and private accounts here, if so, you need to split your author information as well.

With regards to blocking the merge: That's the signature that's not able to be verified, not the linting. The linting isn't marked as "required", so it won't block the merge.

**Pull request merged**
